.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     docker:
       - image: circleci/golang:1.9
 
-    working_directory: /go/src/github.com/owlchain/sebak
+    working_directory: /go/src/github.com/bosnet/sebak
```

Comment:
I found the problem.

This should be changed `boscoin.io/sebak`

Reply:
🤦‍♂️ 
Of course, sorry